docs/13_0_0/components/colorpicker.md

```diff
@@ -63,11 +133,43 @@ ColorPicker has two modes, default mode is _popup_ and other available option is
 <p:colorPicker value="#{bean.color}" mode="inline"/>
 ```
 
+!> Inline mode only allows 1 instance of the inline ColorPicker on the page and you cannot mix Inline and Popup on the same page.  It is a limitation of Coloris.
```

Comment:
This is the first time Coloris is mentioned. Maybe, at the top, mention it's based on Coloris and include a link?

Reply:
Good idea I also updated NOTICE.txt